modules/planning/open_space/coarse_trajectory_generator/hybrid_a_star.cc

```diff
@@ -113,6 +114,10 @@ bool HybridAStar::ValidityCheck(std::shared_ptr<Node3d> node) {
       for (const common::math::LineSegment2d& linesegment :
            obstacle_linesegments) {
         if (bounding_box.HasOverlap(linesegment)) {
+          ADEBUG << "collision start at x: " << linesegment.start().x();
+          ADEBUG << "collision start at y: " << linesegment.start().y();
+          ADEBUG << "collision end at x: " << linesegment.end().x();
+          ADEBUG << "collision end at y: " << linesegment.end().y();
```

Comment:
I checked the implementation of LineSegment2d. The usage here is correct :)

But actually it should not be named as `end()`, which generally means a out-of-bound iterator/object in C++. It should use `last()` or `back()` IMO.